web/package.json

```diff
@@ -27,6 +27,7 @@
     "copy-to-clipboard": "^3.3.1",
     "core-js": "^3.25.0",
     "craco-less": "^2.0.0",
+    "crypto-js": "^4.2.0",
```

Comment:
Will this new dependency impact the yarn build memory usage/build time? Test it, give result